app/models/assignment.rb

```diff
@@ -9,7 +9,7 @@ class Assignment < ActiveRecord::Base
 
   has_many :rubric_criteria,
            -> { order(:position) },
-           class_name: 'RubricCriterion', 
+           class_name: 'RubricCriterion',
```

Comment:
When I put double quotes, it tells me "Prefer single-quoted strings when you don't need string interpolation or special symbols.". So which quotations do i use?